05/app.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { createRoot } from 'react-dom/client';
+import Nav from "./components/Nav";
+
+const root =
+createRoot(document.querySelector('#root'));
+
+const App = () => {
+  const menuItems = [
+    { text: "strona główna", url: "/" },
+    { text: "www", url: "/www" },
+  ];
+
+  return (
+    <div>
+      <h1>My App</h1>
+      <Nav menuItems={menuItems} />
+    </div>
+  );
+};
+
```

Comment:
Brakuje: `root.render(<App />);`

PS. Dobrym zwyczajem jest sprawdzić przed oddaniem czy wszystko działa ;P (na rekrutacji zazwyczaj nikt się nie pochyla nad tym dlaczego nie działa - nie działa to wypadasz :P)